vault/core_metrics.go

```diff
@@ -565,3 +571,55 @@ func (c *Core) inFlightReqGaugeMetric() {
 	// Adding a gauge metric to capture total number of inflight requests
 	c.metricSink.SetGaugeWithLabels([]string{"core", "in_flight_requests"}, float32(totalInFlightReq), nil)
 }
+
+func (c *Core) availablePoliciesGaugeCollector(ctx context.Context) ([]metricsutil.GaugeLabelValues, error) {
+	if c.policyStore == nil {
+		return []metricsutil.GaugeLabelValues{}, nil
+	}
+
+	c.stateLock.RLock()
+	policyStore := c.policyStore
+	c.stateLock.RUnlock()
+
+	ctx = namespace.RootContext(ctx)
+	namespaces := c.collectNamespaces()
+
+	policyTypes := []PolicyType{
+		PolicyTypeACL,
+		PolicyTypeRGP,
+		PolicyTypeEGP,
+	}
+	var values []metricsutil.GaugeLabelValues
+
+	filterDefaultPolicies := func(policies []string) []string {
+		var p []string
+		for _, s := range policies {
+			if s != "default" {
+				p = append(p, s)
+			}
+		}
+		return p
+	}
+
+	for _, pt := range policyTypes {
+		policies, err := policyStore.ListPoliciesForNamespaces(ctx, pt, namespaces)
+		if err != nil {
+			return []metricsutil.GaugeLabelValues{}, err
+		}
+
+		// Filter the "default" policies
+		policies = filterDefaultPolicies(policies)
```

Comment:
Why filter default policies? The default policy is visible in the API/UI when listing policies, so by omitting it, the count will not match what the user can observe in other places.